src/pages/AppList/FiltersAndSorts.ts

```diff
@@ -84,7 +83,6 @@ export namespace AppListFilters {
   };
 
   export const availableFilters: FilterType[] = [
-    NamespaceFilter.create(),
```

Comment:
Same here: below the const `namespaceFilter` looks wrong now, I guess it should be deleted.